packages/seacas/CMakeLists.txt

```diff
@@ -31,6 +31,15 @@ ENDIF ()
 set(SEACASExodus_ENABLE_STATIC_DEFAULT "OFF")
 set(SEACASExodus_ENABLE_SHARED_DEFAULT "OFF")
 
+# Certain environments require either C or CXX linkers for these C unit tests
```

Comment:
The linker language is already specified in the Exodus unit tests in `packages/seacas/libraries/exodus/test/CMakeLists.txt`:
```
  FOREACH ( testName ${cbind_OLDTESTS} )
    TRIBITS_ADD_EXECUTABLE( ${testName} NOEXEPREFIX NOEXESUFFIX SOURCES ${testName}.c LINKER_LANGUAGE CXX)
    set_property(TARGET ${testName} PROPERTY C_STANDARD 99)
  ENDFOREACH ( testName )
```
Should this symbol be used there instead of what is already there, or should similar code just be added to the nemesis test CMakeLists.txt